### PR TITLE
add support for unassociated GraphQL types

### PR DESCRIPTION
**Pull request by Aweary**
This should do the trick. All you need to do is use `typeMapper` to associate the name and GraphQL type and it will correctly resolve it when using the `nodeInterface` and `nodeField` methods

Resolves https://github.com/mickhansen/graphql-sequelize/issues/30

### CODE REVIEW

File: test/relay.test.js [hunk 1]
```diff
@@ -136,10 +137,24 @@ describe('relay', function () {
...
+      }),
+      interfaces: [nodeInterface]
+});
+
     nodeTypeMapper.mapTypes({
       [User.name]: userType,
       [Project.name]: projectType,
-      [Task.name]: taskType
+      [Task.name]: taskType,
+      viewer: viewerType
```
Comment (mickhansen) [inline test/relay.test.js]: I'm not totally happy with the uncased matching, but i doubt people willa ctually have types with different cases of the same name - But if it does happen we'll break it later.

Comment (Aweary) [inline test/relay.test.js]: Yeah, it's already normalizing the types here https://github.com/mickhansen/graphql-sequelize/blob/master/src/relay.js#L25 but I think its safe for right now. Not a huge issue changing that.

**Comment by Aweary** [comment]
@brad-decker is currently out of the office for the week, so he may be slow to respond on this. Just a heads up.

**Comment by mickhansen** [comment]
Alright, would like his feedback since he implemented it originally - How urgently do you need it merged?

**Comment by Aweary** [comment]
Not a big rush, I'm just pulling `graphql-sequelize` from our fork instead of `npm`, so we can wait it out. :+1:

**Comment by mickhansen** [comment]
@brad-decker Back yet? :)

**Comment by Aweary** [comment]
@brad-decker is back, and we discussed the casing coercion and decided that we should just remove it. So model/type names are case sensitive.

**Comment by brad-decker** [comment]
Sorry for the delay @mickhansen

**Comment by mickhansen** [comment]
@brad-decker Not a problem, just wanted to ping :)

**Comment by Aweary** [comment]
:tada: :tada: :tada:

**Comment by mickhansen** [comment]
`v0.11.0`

**Pull request merged**